docs/content/docs/libs/state_processor_api.md

```diff
@@ -515,6 +515,28 @@ Before you interrogate state using the table API, make sure to review our [Flink
 
 IMPORTANT NOTE: State Table API only supports keyed state.
 
+### Metadata
+
+The SQL connector allows users to read metadata of savepoints in the following way:
+```SQL
+LOAD MODULE state;
+SELECT * FROM savepoint_metadata('/root/dir/of/checkpoint-data/chk-1');
+```
+
+The table have the following fixed schema:
+
+| Key                                      | Data type       | Description                                                                      |
+|------------------------------------------|-----------------|----------------------------------------------------------------------------------|
+| checkpoint-id                            | BIGINT NOT NULL | Checkpoint ID.                                                                   |
+| operator-name                            | STRING          | Operator Name.                                                                   |
+| operator-uid                             | STRING          | Operator UID.                                                                    |
+| operator-uid-hash                        | STRING NOT NULL | Operator UID hash.                                                               |
+| operator-parallelism                     | INT NOT NULL    | Parallelism of the operator.                                                     |
+| operator-max-parallelism                 | INT NOT NULL    | Maximum parallelism of the operator.                                             |
+| operator-subtask-state-count             | INT NOT NULL    | Number of operator subtask states.                                               |
```

Comment:
`operator-subtask-state-count` is an addition compared to the FLIP. I thought it's valuable information but if somebody has objection we can remove it.

Reply:
It looks like a good addition to me